libraries/chain/controller.cpp

```diff
@@ -1196,8 +1196,8 @@ struct controller_impl {
                      ("lib_num", lib_num)("bn", fork_db_root_block_num()) );
       }
 
-      const uint32_t hs_lib = hs_irreversible_block_num;
-      const uint32_t new_lib = hs_lib > 0 ? hs_lib : fork_db_head_irreversible_blocknum();
+      const uint32_t if_lib = if_irreversible_block_num;
+      const uint32_t new_lib = if_lib > 0 ? if_lib : fork_db_head_irreversible_blocknum();
```

Comment:
This `if_lib > 0` really looks like a regular `if` statement😺 We can try to reduce `if_` prefix.

Reply:
What about we standardize on `inf`: `inf_lib`, `set_inf_irreversible_block_number` ? @greg7mdp @linh2931

Reply:
@arhag mentioned we should try to avoid `instant` and just say `finality`.

Reply:
We can clean up those later.

Reply:
I'm OK with `inf_lib`, although I thought `if_lib` was fine too, and I don't think `inf` is clearer.
But then I think the set function should be `set_inf_lib`.